CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - OTLP .proto files from `open-telemetry/opentelemetry-proto` imported as a git submodule under `internal/opentelemetry-proto`.
    References to `github.com/open-telemetry/opentelemetry-proto` changed to `go.opentelemetry.io/otel/internal/opentelemetry-proto-gen`. (#948)
 
+### Changed
```

Comment:
Can we break out the specifics of this change set from the end user perspective. These notes are commonly used when upgrading or evaluating releases so it will be good to make it clear the things that were removed or replace.

I.e. 

> Specifically the changeset includes:
> * for Jager exporter, introduced InstallNewPipeline and removed RegisterGlobal option instead
> * for Zipkin exporter, introduced pipeline methods and adjusted the new exporter method